src/app/services/trips.service.ts

```diff
@@ -110,10 +111,14 @@ export class TripsService {
   }
 
 
-  getTrip(tripIndex: string) {
+  getTrip(tripIndex: string): Trip {
     return this.trips.filter((trip) => {
        return trip.id == tripIndex;
     })[0]
   }
 
+  getTrips() {
```

Comment:
@ashish173 as in this PR i have mainly written code for trips list component, so i will change the above function when i will work on trip detail  component.